contributor/making-release.md

```diff
@@ -125,8 +167,12 @@ After a successful vote, [this website][website-repo] needs to be updated with t
   * Create a post in `_posts/release/` containing release notes
   * Remove previous bug-fix release (if applicable)
   * Update doap/accumulo.rdf
+  * complete release notes 
+  * update previous release notes (as archived or archived-critical) 
+  * update _config.yml 
+  * update _includes/nav.html 
```

Comment:
Escape the underscores, because it's a markdown char. Could use backtick to quote the filenames.